### ISSUE TITLE
Northern AZ Market

### ISSUE THREAD

**Issue by sean8300** [ListerAssister/issues#295]
Please create a Northern AZ market in addition to Phoenix & Tucson
Submitted By: Greg

**Comment by sean8300** [comment]
JUVI: Check with Greg and see if this is still needed

**Comment by Juvie92** [comment]
Messaged Greg and he said he will talk to Nicole and get back to me.

**Comment by sean8300** [comment]
Old email from Greg:
Good Morning Sean,

Pricing is the same as the rest of AZ/NV.

Inventory will be stored & managed in Phx until such time as orders/revenue make acquiring storage space in Prescott/Prescott Valley feasible.

I would suggest matching the service area with the zip code list we did with Nicole a few weeks ago.  Please review this with her to determine the service area and update the zip code list if necessary.  Please call my cell if either of you has any questions as I will be driving until early afternoon today and have marketing appointments later this afternoon and evening. 

Thanks,
Greg

**Comment by Juvie92** [comment]
![image](https://user-images.githubusercontent.com/64864069/83783772-1a27ed80-a645-11ea-8236-4e228166f89c.png)

**Comment by sean8300** [comment]
These zips are being overridden in siteConfig.js because we aren't there everyday. I don't know if this is the normal usage or not, so need to know if this is a daily thing or if we are still only visiting certain days. This is on hold until I know

Updating info on ListerAssister/issues#295, waiting on response from Nicole/Greg, will decide what to do after feedback from them.

**Comment by sean8300** [comment]
Slacked Greg/Nicole asking if we are in this area daily

**Comment by sean8300** [comment]
Ok, going to make a change here. 

On pricing_zip_code lets go ahead and add a new bool column "day_restricted", set the zips listed in this ticket to true

On the order form (siteConfig) have it do a search for whether or not day_restricted is true for the zip entered, if it is then display a message along the lines of "We only service this region on particular days".....for now we'll leave it at that, may expand this to actually store the days we service that zip and display that info here.  For now lets just roll with that, am waiting on slack responses on what to do with the other zips listed in zip_array on siteConfig, and what they want us to do about the date selection

**Comment by sean8300** [comment]
Slacked Greg and Nicole again since I haven't heard anything:
Any update on the Northern AZ market zip code questions?
Some of the zips I was given for the North AZ market are in this list. What I'm asking is:
1 - Should these zips be taken out of this list and display a different message, if so, what message
2 - Should we have a list that displays the current message for one range of zips then another message for the northern AZ range

**Comment by sean8300** [comment]
Slacked Nicole and Greg again. This is becoming the usual thing, I have to Slack each one of these two numerous times over the course of a couple weeks and get to the point that I'm pretty much spamming them, then eventually pull one of them aside when they come into the office before I'm even acknowledged in any way.  
Any update on the Northern AZ market zip code questions?
Some of the zips I was given for the North AZ market are in this list. What I'm asking is:
1 - Should these zips be taken out of this list and display a different message, if so, what message
2 - Should we have a list that displays the current message for one range of zips then another message for the northern AZ range

**Comment by sean8300** [comment]
Response from Greg:
86333, 86322, 86329, 86327, 86314, 86301, 86303, 86305

**Comment by sean8300** [comment]
That was an answer to something, not what Ive been asking though....dont know what this is

**Comment by sean8300** [comment]
Any update on the Northern AZ market zip code questions?
Some of the zips I was given for the North AZ market are in this list. What I'm asking is:
1 - Should these zips be taken out of this list and display a different message, if so, what message
2 - Should we have a list that displays the current message for one range of zips then another message for the northern AZ range

**Comment by sean8300** [comment]
What exactly are we doing with this list?

**Comment by sean8300** [comment]
In the system there is a list of zip codes that when entered trigger the order form displays the message "The zip code you have selected is outside our normal service area and must be scheduled through our office.  We will reach out to you while processing your order regarding the potential date of service or you are welcome to contact Customer Service at 602-492-5006 for assistance."
Some of the zips I was given for the North AZ market are in this list.  What I'm asking is:
1 - Should these zips be taken out of this list and display a different message, if so, what message
2 - Should we have a list that displays the current message for one range of zips then another message for the northern AZ range
1:15
this is creating a northern AZ market

**Comment by sean8300** [comment]
The zip codes we gave you should get a 50$ charge with no message

**Comment by sean8300** [comment]
Finally got a response, it took asking the same question five times over two weeks to finally get an answer, and in that answer I am made aware of an additional $50 for the first time

**Comment by sean8300** [comment]
taking over from Juvi, already spoke to him about it so hes aware that I'm taking this

**Comment by sean8300** [comment]
Full list of all zips I've been given to this point:
86333, 86322, 86329, 86327, 86314, 86301, 86303, 86305, 86301, 86302, 86303, 86304, 86305, 86313

**Comment by Juvie92** [comment]
No longer going to go with this. Closing.
